packages/@aws-cdk/aws-cognito-identitypool/test/identitypool.test.ts

```diff
@@ -211,6 +211,15 @@ describe('identity pool', () => {
     expect(idPool.identityPoolArn).toMatch(/cognito-identity:my-region:1234567891011:identitypool\/my-region:idPool/);
   });
 
+  test('user pools auth provider properly generates URL', () => {
+    const stack = new Stack();
+    const pool = new UserPool(stack, 'Pool');
+    const authProvider = new UserPoolAuthenticationProvider({ userPool: pool });
+    console.log(Template.fromStack(stack));
```

Comment:
Just in case our error message isn't clear, this line is causing the build failure right now.